components/ActivityForm/index.js

```diff
@@ -0,0 +1,159 @@
+import styled from "styled-components";
+import { StyledButton } from "@/components/StyledButton";
+import { useRouter } from "next/router";
+import { StyledBackLink } from "@/components/StyledLinks";
+import { Icon } from "@/components/Icon";
+
+export default function ActivityForm({ onAddActivity }) {
+  const router = useRouter();
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const newActivity = Object.fromEntries(formData);
+    newActivity.category = formData.getAll("category");
+    if (newActivity.category.length === 0) {
+      alert("Please select at least one category.");
+      return false;
+    }
+    onAddActivity(newActivity);
+    router.push("/");
+  }
+
+  return (
+    <>
+      <StyledBackLink href={"/"}>
+        <Icon name="chevronLeft" />
+        Back to all Activities
+      </StyledBackLink>
+
+      <StyledForm onSubmit={handleSubmit}>
+        <StyledLabel htmlFor="title">Activity</StyledLabel>
+        <StyledInput
+          id="title"
+          name="title"
+          type="text"
+          maxLength="150"
+          placeholder="add activity-title"
+          required
+        />
+        <StyledLabel htmlFor="area">Area</StyledLabel>
+        <StyledInput
+          id="area"
+          name="area"
+          type="text"
+          placeholder="add area"
+          required
+        />
+        <StyledLabel htmlFor="country">Country</StyledLabel>
+        <StyledInput
+          id="country"
+          name="country"
+          type="text"
+          placeholder="add country"
+          required
+        />
+        <StyledLabel htmlFor="category">Category</StyledLabel>
+        <StyledCheckboxContainer>
```

Comment:
One more thing comes into my mind. Just a suggestion you can use or not:
It could be a good idea to have an array called `categories` in your `lib/dummyData.js`. Then you can import and use it here like:
```js
categories.map(category=><label><StyledInput id={category.name} name="category" ... /><label>)
```

This way you can easily add or remove categories in the future without changing the codebase.